server/apis/v1/handler.go

```diff
@@ -82,14 +83,40 @@ func NewHandler() (*handler, error) {
 	}, nil
 }
 
+// Login is used to redirect the user to authentication page and set the user identity token in the cookie
+func (h *handler) Login(c *gin.Context) {
+	// TODO - send a request to Dex to get the real user identity token.
+	token := "dummy-token"
+	c.SetCookie("user-identity-token", token, 3600, "/", "", true, true)
+	returnUrl := c.Request.URL.Query().Get("returnUrl")
+	c.JSON(http.StatusOK, returnUrl)
```

Comment:
I think API needs to do http.redirect here so UI is redirected back to where it came from